tests/net/lib/coap/src/main.c

```diff
@@ -1110,14 +1130,14 @@ ZTEST(coap, test_build_options_out_of_order_0)
 			  "Built packet doesn't match reference packet");
 }
 
-#define ASSERT_OPTIONS(cpkt, expected_opt_len, expected_data, expected_data_len)                   \
-	do {                                                                                       \
-		static const uint8_t expected_hdr_len = 9;                                         \
-		zassert_equal(expected_hdr_len, cpkt.hdr_len, "Wrong header length");              \
-		zassert_equal(expected_opt_len, cpkt.opt_len, "Wrong option length");              \
-		zassert_equal(expected_hdr_len + expected_opt_len, cpkt.offset, "Wrong offset");   \
-		zassert_equal(expected_data_len, cpkt.offset, "Wrong offset");                     \
-		zassert_mem_equal(expected_data, cpkt.data, expected_data_len, "Wrong data");      \
+#define ASSERT_OPTIONS(cpkt, expected_opt_len, expected_data, expected_data_len)		\
+	do {											\
+		static const uint8_t expected_hdr_len = 9;					\
+		zassert_equal(expected_hdr_len, cpkt.hdr_len, "Wrong header length");		\
+		zassert_equal(expected_opt_len, cpkt.opt_len, "Wrong option length");		\
+		zassert_equal(expected_hdr_len + expected_opt_len, cpkt.offset, "Wrong offset");\
+		zassert_equal(expected_data_len, cpkt.offset, "Wrong offset");			\
+		zassert_mem_equal(expected_data, cpkt.data, expected_data_len, "Wrong data");	\
```

Comment:
This looks like unrelated cosmetic change if I managed to read the lines correctly. Please remove this from the commit if so. If you really want to change this, place the change to new commit in this PR.